pkg/label/filter.go

```diff
@@ -0,0 +1,96 @@
+package label
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/Peripli/service-manager/pkg/types"
+
+	"github.com/Peripli/service-manager/pkg/query"
+	"github.com/Peripli/service-manager/pkg/util"
+	"github.com/Peripli/service-manager/pkg/web"
+	"github.com/tidwall/gjson"
+)
+
+// DenyLabelOperations describe denied label key operations
+type DenyLabelOperations map[string][]query.LabelOperation
+
+type FilterOperations struct {
+	deniedOperations DenyLabelOperations
+}
+
+func NewFilterOperations(deniedOperations DenyLabelOperations) *FilterOperations {
+	return &FilterOperations{
+		deniedOperations: deniedOperations,
+	}
+}
+
+func (flo *FilterOperations) Name() string {
+	return "FilterOperations"
```

Comment:
Not sure if the filter name is descriptive enough

Reply:
ForbiddenLabelOperationsFilter ?

Reply:
Sounds good